### PR TITLE
#162725224 users should be able to rate articles

### PR DESCRIPTION
**Pull request by malfahad**
#### What does this PR do?
Authenticated users should be able to rate articles.

#### Description of Task to be completed?
* A 5-star rating system is the end goal. Let's have authenticated users get visual feedback of the average rating of their articles. 

#### How should this be manually tested?
* Fetch and checkout into this branch with `git fetch origin ft-rate-articles-162725224 && git checkout ft-rate-articles-162725224`
* Install dependencies and start the app with `yarn install && yarn start`
* Login to authors haven
* Select a single article, view it and then use the stars to rate an article.

#### What are the relevant pivotal tracker stories?
[#162725224](https://www.pivotaltracker.com/n/projects/2232225/stories/162725224)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ import './assets/style.scss';
 import '../node_modules/argon-design-system-free/assets/css/argon.css';
 import './assets/css/socialAuth.css';
 import './assets/css/comments.css';
-import 'react-toastify/dist/ReactToastify.css';
```
Comment (malfahad) [inline src/index.js:12]: this was a duplicate import

**Comment by MuhweziDeo** [comment]
Change the average rating from stars to a figure @malfahad

**Review by MuhweziDeo — DISMISSED** [review]
LGTM

**Comment by RrNn** [comment]
@malfahad , I can't seem to rate an article. They seem to be rated already.

**Comment by malfahad** [comment]
@RrNn , Please login with a different account

**Pull request merged**